epinorm/dataFetcher.py

```diff
@@ -0,0 +1,100 @@
+import pandas as pd
+from polyglot.text import Text
+from transliterate import translit
+
+def getCountryNamesStandardised():
+    """
+    THis retrieves the country names in the file "countries", as they are in the file
+    """
+
+    df = pd.read_csv("epinorm/data/countries.csv")
+    names = df["name"]
+    return set(names)
+
+def getAdminLevelsOfCountryTable():
+
+    # * create a mapping from country name to its code
+    countryToCode = {}
+
+    df = pd.read_csv("epinorm/data/countries.csv")
+    for _, row in df.iterrows():
+        countryToCode[row["name"]] = row["alpha_2"]
+
+    # there are exceptions
```

Comment:
How about modifying this directly in the source file `epinorm/data/countries.csv` ?
Or is it important to keep this file as is ?